### PR TITLE
Tn/add project vars

### PR DESCRIPTION
**Pull request by TylerNoblett**
This PR adds project specific vars that can be accessed by a super admin. This is one of three PRs to address this [request](https://github.com/ashirt-ops/ashirt-server/issues/864); the next will add project specific variables and the final one will send both sets of variables to service workers
Screen Shot 2023-09-27 at 11 41 31 AM

Notes
- only a super admin can create, edit and delete project variables
- variables aren't required to have a value
- variable names are unique

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: backend/database/seeding/hp_seed_data.go [hunk 1]
```diff
@@ -361,3 +364,11 @@ var FindingBook2Robes = newHPFinding(OpChamberOfSecrets.ID, "find-uuid-robes", n
...
 var newHPServiceWorker = newServiceWorkerGen(1)
 var DemoServiceWorker = newHPServiceWorker("Demo", `{ "type": "web",  "version": 1, "url": "http://demo:3001/process" }`)
+
+var newGlobalVar = newGlobalVarGen(1)
+var VarExpelliarmus = newGlobalVar("Expelliarmus", "disarm an opponent")
+var VarAlohomora = newGlobalVar("Alohomora", "unlock doors")
+var VarAscendio = newGlobalVar("Ascendio", "lifts the caster high into the air")
+var VarImperio = newGlobalVar("Imperio", "control another person")
+var VarLumos = newGlobalVar("Lumos", "creates a narrow beam of light")
+var VarObliviate = newGlobalVar("Obliviate", "erases memories")
```
Comment (JoelAtDeluxe) [inline backend/database/seeding/hp_seed_data.go:374]: :+1:

File: backend/services/global_vars_test.go [hunk 2]
```diff
@@ -0,0 +1,160 @@
+// Copyright 2022, Yahoo Inc.
```
Comment (JoelAtDeluxe) [inline backend/services/global_vars_test.go:1]: Maybe it's worth updating the copyright?

Comment (TylerNoblett) [reply backend/services/global_vars_test.go:1]: whoops! good catch

File: backend/database/seeding/hp_seed_data.go (hunk 1 above)
Comment (TylerNoblett) [reply backend/database/seeding/hp_seed_data.go:374]: haha I'm all about 1) honoring convention and 2) referencing media franchises from my childhood in my work

**Review by JoelAtDeluxe — COMMENTED** [review]
Still looking at the frontend code, but the backend looks good.

**Comment by TylerNoblett** [comment]
whoops sorry Joel; I've updated a few things since you last looked - I second guessed some decisions as I was copying a bunch of this stuff over to create operation variables

File: backend/migrations/20230922175734-add-global-vars.sql [hunk 3]
```diff
@@ -0,0 +1,12 @@
+-- +migrate Up
+CREATE TABLE global_vars (
+  id INT AUTO_INCREMENT,
+  name VARCHAR(255) NOT NULL UNIQUE,
+  value VARCHAR(255) NOT NULL,
```
Comment (jkennedyvz) [inline backend/migrations/20230922175734-add-global-vars.sql:5]: Is VARCHAR(255) large enough? What if* I wanted to pass an RSA key or longer document?

Comment (TylerNoblett) [reply backend/migrations/20230922175734-add-global-vars.sql:5]: oh that's a good point - yeah I can def change that

Comment (TylerNoblett) [reply backend/migrations/20230922175734-add-global-vars.sql:5]: changed the type to TEXT - how does that sound?

Comment (TylerNoblett) [reply backend/migrations/20230922175734-add-global-vars.sql:5]: just tested by adding a really long doc and the UI 'breaks' so I'll fix that as well

**Review by jkennedyvz — APPROVED** [review]

**Pull request merged**